game.go

```diff
@@ -737,6 +738,11 @@ func (p *Game) doWhenLeftButtonDown(ev *eventLeftButtonDown) {
 			}
 		}
 	}
+
+	if p.inputs.canTriggerClickEvent(inputStageClickTimerId) {
```

Comment:
What if some sprite is clicked, while `p.inputs.canTriggerClickEvent(syncSprite.GetId())` returns false because of `inputMouseClickIntervalMs`? The stage-click will be triggered, which I think is not expected.